libraries/engage/locations/components/Cart/CartChangeFulfillmentMethod.connector.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import * as Redux from 'redux';
+import { connect } from 'react-redux';
+import { updateProductsInCart } from '@shopgate/engage/cart';
+import { fetchProductLocations } from '../../actions';
+import {
+  PRODUCT_FULFILLMENT_METHOD_DIRECT_SHIP,
+  PRODUCT_FULFILLMENT_METHOD_ROPIS,
+} from '../../constants';
+import { type OwnProps, type DispatchProps } from './CartItemProductChangeLocation.types';
+
+/**
+ * @param {Function} dispatch The redux dispatch function.
+ * @return {Object} The extended component props.
+ */
+const mapDispatchToProps = (dispatch: Redux.Dispatch<any>): DispatchProps => ({
+  fetchProductLocations,
+  updateProductInCart: (cartItemId, quantity, method, location) => {
+    if (method === PRODUCT_FULFILLMENT_METHOD_DIRECT_SHIP) {
+      dispatch(updateProductsInCart({
```

Comment:
`updateProductsInCart` accept Array but Object is actually given

Reply:
Is it even used inside of component?

Reply:
You are right, it is not used. Removed it.